src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -340,3 +361,20 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig replica create') as c:
         c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location at which to create the replica.')
+
+    with self.argument_context('appconfig snapshot') as c:
+        c.argument('snapshot_name', arg_type=snapshot_name_arg_type)
+
+    with self.argument_context('appconfig snapshot create') as c:
+        c.argument('filters', arg_type=snapshot_filter_arg_type)
+        c.argument('composition_type', arg_type=get_enum_type(["key", "key_label"]), help='Composition type used in building App Configuration snapshots. If not specified, defaults to key.')
+        c.argument('retention_period', type=int, help='Duration in seconds for which a snapshot can remain archived before expiry. A snapshot can be archived for a maximum of 7 days (604,800s) for free tier stores and 90 days (7,776,000s) for standard tier stores. If specified, retention period must be at least 1 hour (3600s)')
+        c.argument('tags', arg_type=tags_type, help="Space-separated tags: key[=value] [key[=value] ...].")
```

Comment:
The help message has already been defined in `tags_type`, so there is no need to repeat the definition
```suggestion
        c.argument('tags', arg_type=tags_type)
```

Reply:
![image](https://github.com/Azure/azure-cli/assets/102393878/dcc3b0a5-54d2-425d-b7d5-ecf0776054c5)
@zhoxing-ms Please the original help message has additional text that is not applicable to the parameter. i.e. `Use "" to clear existing tags.`. This is why we indicate our own custom message to override it.

Reply:
May I ask why this parameter dose not support using "" to clear existing tags?

Reply:
This command is a `create` command, and will only succeed if a resource with the specified name does not already exist, so no replacement is possible for this command.